gamemodes/terrortown/gamemode/shared/sh_equip_items.lua

```diff
@@ -836,11 +838,44 @@ local function InitDefaultEquipmentForRole(roleData)
 	roleData.fallbackTable = tbl
 end
 
+local function CleanUpDefaultCanBuyIndices()
+	local itms = items.GetList()
+
+	-- load items
+	for i = 1, #itms do
+		local itm = itms[i]
+		local CanBuy = {}
+		itm.CanBuy = itm.CanBuy or {}
```

Comment:
Maybe a small converting function would be useful. If you wanna save performance, then pls add a comment too :)

Reply:
Done